web-ui/controller/index.ml

```diff
@@ -1,21 +1,21 @@
 module Client = Ocaml_ci_api.Client
 
-let ( >>!= ) = Git_forge.( >>!= )
+let ( >>!!= ) = Git_forge.( >>!= )
+let ( >|!= ) a f = Lwt_result.map f a
```

Comment:
This is too much. I'd suggest switching to the `Syntax` module instead.
Are you trying to ascii-art spaceships?

Reply:
Kind of :smile: